dart/example/event_example.dart

```diff
@@ -5,15 +5,15 @@ final event = SentryEvent(
   serverName: 'server.dart',
   release: '1.4.0-preview.1',
   environment: 'Test',
-  message: Message('This is an example Dart event.'),
+  message: SentryMessage('This is an example Dart event.'),
   tags: const <String, String>{'project-id': '7371'},
   extra: const <String, String>{'section': '1'},
-  fingerprint: const <String>['example-dart'],
-  user: User(
+  // fingerprint: const <String>['example-dart'],
```

Comment:
its nice to test and see how events are grouped by default and not forcing a `fingerprint`

Reply:
Same here.